config/config.exs

```diff
@@ -22,6 +22,11 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:user_id]
 
+# Guardian config
+config :dml, Dml.Guardian,
+  issuer: "DML",
+  secret_key: "u49DGdIfepFS21VX6A30S7K32KZOvm54SL/AsvCuSSN1UJM4K6fMR8ImEkmAqcJ5"
```

Comment:
This a production key?

Reply:
It's a development key, on production it will be loaded from and ENV variable 👍